catalog_full_include.html

```diff
@@ -0,0 +1,27 @@
+{% for c in core %}
```

Comment:
Is this why we need core classes? So we can list them at the start?  There might be other, more general purpose ways to do this.